build.sbt

```diff
@@ -108,12 +108,6 @@ lazy val demoSettings       = Seq(
           |      },
           |    },
           |    build: {
-          |      minify: 'terser',
```

Comment:
Is this to better debug the demo projects?

Reply:
No I assume this was copied from explore. But terser is not installed in this project, and minification is done by default with esbuild (which is much faster). So before `npx vite build` actually didn't work